### PR TITLE
[FIX] update some internal links

### PR DESCRIPTION
**Pull request by Remi-Gau**
- closes https://github.com/bids-standard/bids-specification/issues/2003

### CODE REVIEW

**Comment by Remi-Gau** [comment]
silly auto reformat gives you a huge diff... grrr

File: src/derivatives/imaging.md [hunk 1]
```diff
@@ -14,20 +14,19 @@ Template:
...
-or surface sampling density being the only difference between them, then the
-[`res`](../appendices/entities.md#res) (for *resolution* of regularly sampled N-D data) and/or
-[`den`](../appendices/entities.md#den) (for *density* of non-parametric surfaces)
-entities SHOULD be used to avoid name conflicts.
-Note that only files combining both regularly sampled (for example, gridded)
-and surface sampled data (and their downstream derivatives) are allowed
-to present both [`res`](../appendices/entities.md#res) and
+When two or more instances of a given derivative are provided with resolution or
+surface sampling density being the only difference between them, then the
+[`res`](../appendices/entities.md#res) (for _resolution_ of regularly sampled
```
Comment (effigies) [inline src/derivatives/imaging.md:24]: I take it you just ran remark? It doesn't like its own output. I hope you still have the pre-reformat changes in your buffer.

**Review by effigies — APPROVED** [review]
Merge if you're ready.

**Comment by Remi-Gau** [comment]
most errors left are related to subobects rendering: won't fix here

https://app.readthedocs.org/projects/bids-specification/builds/26570977/#253898759--15

```
INFO    -  Doc file 'modality-agnostic-files.md' contains a link './glossary.md#objects.subobject.Name', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Name'.
INFO    -  Doc file 'modality-agnostic-files.md' contains a link './glossary.md#objects.subobject.Version', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Version'.
INFO    -  Doc file 'modality-agnostic-files.md' contains a link './glossary.md#objects.subobject.Description', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Description'.
INFO    -  Doc file 'modality-agnostic-files.md' contains a link './glossary.md#objects.subobject.CodeURL', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodeURL'.
INFO    -  Doc file 'modality-agnostic-files.md' contains a link './glossary.md#objects.subobject.Container', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Container'.
INFO    -  Doc file 'appendices/entity-table.md' contains a link '../glossary.md#objects.suffixes.PLI', but the doc 'glossary.md' does not contain an anchor '#objects.suffixes.PLI'.
INFO    -  Doc file 'appendices/entity-table.md' contains a link '../glossary.md#objects.suffixes.2PE', but the doc 'glossary.md' does not contain an anchor '#objects.suffixes.2PE'.
INFO    -  Doc file 'modality-specific-files/genetic-descriptor.md' contains a link '../glossary.md#objects.subobject.Dataset', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Dataset'.
INFO    -  Doc file 'modality-specific-files/genetic-descriptor.md' contains a link '../glossary.md#objects.subobject.Database', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Database'.
INFO    -  Doc file 'modality-specific-files/genetic-descriptor.md' contains a link '../glossary.md#objects.subobject.Descriptors', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Descriptors'.
INFO    -  Doc file 'modality-specific-files/magnetic-resonance-imaging-data.md' contains a link '#in-and-out-plane-spatial-encoding', but there is no such anchor on this page.
INFO    -  Doc file 'modality-specific-files/magnetic-resonance-imaging-data.md' contains a link '../glossary.md#objects.subobject.CodeValue', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodeValue'.
INFO    -  Doc file 'modality-specific-files/magnetic-resonance-imaging-data.md' contains a link '../glossary.md#objects.subobject.CodeMeaning', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodeMeaning'.
INFO    -  Doc file 'modality-specific-files/magnetic-resonance-imaging-data.md' contains a link '../glossary.md#objects.subobject.CodingSchemeDesignator', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodingSchemeDesignator'.
INFO    -  Doc file 'modality-specific-files/magnetic-resonance-imaging-data.md' contains a link '../glossary.md#objects.subobject.CodingSchemeVersion', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodingSchemeVersion'.
INFO    -  Doc file 'modality-specific-files/microscopy.md' contains a link '../glossary.md#objects.suffixes.PLI', but the doc 'glossary.md' does not contain an anchor '#objects.suffixes.PLI'.
INFO    -  Doc file 'modality-specific-files/positron-emission-tomography.md' contains a link '../glossary.md#objects.subobject.CodeValue', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodeValue'.
INFO    -  Doc file 'modality-specific-files/positron-emission-tomography.md' contains a link '../glossary.md#objects.subobject.CodeMeaning', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodeMeaning'.
INFO    -  Doc file 'modality-specific-files/positron-emission-tomography.md' contains a link '../glossary.md#objects.subobject.CodingSchemeDesignator', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodingSchemeDesignator'.
INFO    -  Doc file 'modality-specific-files/positron-emission-tomography.md' contains a link '../glossary.md#objects.subobject.CodingSchemeVersion', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.CodingSchemeVersion'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.OperatingSystem', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.OperatingSystem'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.ScreenDistance', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.ScreenDistance'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.ScreenRefreshRate', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.ScreenRefreshRate'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.ScreenResolution', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.ScreenResolution'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.ScreenSize', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.ScreenSize'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.SoftwareName', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.SoftwareName'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.SoftwareRRID', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.SoftwareRRID'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.SoftwareVersion', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.SoftwareVersion'.
INFO    -  Doc file 'modality-specific-files/task-events.md' contains a link '../glossary.md#objects.subobject.Code', but the doc 'glossary.md' does not contain an anchor '#objects.subobject.Code'.
```

**Comment by Remi-Gau** [comment]
still not sure about PLE and PI suffixes error but that will be for another day

**Comment by effigies** [comment]
> INFO    -  Doc file 'appendices/entity-table.md' contains a link '../glossary.md#objects.suffixes.PLI', but the doc 'glossary.md' does not contain an anchor '#objects.suffixes.PLI'.
> INFO    -  Doc file 'appendices/entity-table.md' contains a link '../glossary.md#objects.suffixes.2PE', but the doc 'glossary.md' does 
> INFO    -  Doc file 'modality-specific-files/microscopy.md' contains a link '../glossary.md#objects.suffixes.PLI', but the doc 'glossary.md' does not contain an anchor '#objects.suffixes.PLI'.

These are all fixable. `#pli-suffix` and `#twope-suffix`.

**Comment by Remi-Gau** [comment]
> These are all fixable. `#pli-suffix` and `#twope-suffix`.

but I think they are generated by macros, no?

**Pull request merged**